kroxylicious-operator/examples/simple/02.server-certificate.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: server-certificate
+  namespace: my-proxy
+spec:
+  isCA: true
```

Comment:
Is this requesting a CA cert? If so, I don't think we need that.

Reply:
agree - that shouldn't be required.